driver/src/main/scala/org/virtuslab/ideprobe/pants/PantsPluginBuilder.scala

```diff
@@ -35,11 +37,17 @@ object PantsPluginBuilder extends DependencyBuilder(Id("pants")) {
     if (result.exitCode != 0) throw new Exception(s"Couldn't build pants plugin. STDERR:\n${result.err}")
 
     val files = localRepo.toFile.listFiles()
-    val pantsPath = files.find(_.getName.matches("pants_.*\\.zip")).getOrElse {
+    val output = files.find(_.getName.matches("pants_.*\\.zip")).getOrElse {
       throw new Exception(s"Couldn't find pants archive. Existing files:\n${files.mkString("\n")}")
     }
-    println(s"Built pants at $pantsPath")
-    pantsPath.toPath.inputStream
+
+    val pantsPath = Paths.get("/tmp", "pants.zip")
+    output.renameTo(pantsPath.toFile)
+    if(java.nio.file.Files.exists(pantsPath)) {
```

Comment:
It can also fail due to a number of other problems (e. g. incorrect permissions or no space left). 

On the other hand, the logic of moving the archive is somewhat unreliable by design, because it involves guessing of the file name. If the guess is wrong for any reason (e. g. the naming convention in the Pants build scripts has changed), I'd like to know what files are present, so that it's easier to correct the guess.